src/bentoml/testing/grpc/__init__.py

```diff
@@ -39,17 +40,38 @@
 ]
 
 
-def randomize_pb_ndarray(shape: tuple[int, ...]) -> pb.NDArray:
+def create_bento_servicer(
```

Comment:
Wouldn't it make more sense to just take two arguments instead of returning a function?

Reply:
yes